easybuild/easyblocks/f/fftw.py

```diff
@@ -181,8 +183,20 @@ def run_all_steps(self, *args, **kwargs):
                 if (prec == 'single' and (self.asimd or self.neon)) or (prec == 'double' and self.asimd):
                     prec_configopts.append('--enable-neon')
 
+                # For POWER with GCC 5/6/7 and FFTW/3.3.6 we need to disable some settings for tests to pass
+                # (we do it last so as not to affect previous logic)
+                cpu_arch = get_cpu_architecture()
+                fftw_version = self.version
+                if cpu_arch in [POWER] and self.toolchain.comp_family() in [TC_CONSTANT_GCC] and \
+                        LooseVersion(fftw_version) == LooseVersion("3.3.6"):
+                    # See https://github.com/FFTW/fftw3/issues/59 which applies to GCC 5/6/7
+                    if (prec == 'single'):
+                        prec_configopts.append('--disable-altivec')
+                    if (prec == 'double'):
+                        prec_configopts.append('--disable-vsx')
+
                 # append additional configure options (may be empty string, but that's OK)
-                self.cfg.update('configopts', [' '.join(prec_configopts) + common_config_opts])
+                self.cfg.update('configopts', [' '.join(prec_configopts) + ' ' + common_config_opts])
```

Comment:
Since this is a bug fix, maybe this should be done in a separate PR? Trivial to merge...

Reply:
Done in #1277

Reply:
merged #1277, thanks